app/src/org/commcare/tasks/ManageKeyRecordTask.java

```diff
@@ -410,6 +409,7 @@ private void markOldRecordsInactive(SqlStorage<UserKeyRecord> storage, String us
         for (UserKeyRecord r : allRecordsWithSameUsername) {
             if (!r.getUuid().equals(uuidOfActiveRecord)) {
                 r.setInactive();
+                storage.write(r);

```

Comment:
Bug fix that should be double checked. I'm not sure if this was causing things to break or not, but it sorta sketched me out. `r.setInactive` was being set, but the change wasn't being committed to storage. 

Original change was made in https://github.com/dimagi/commcare-android/pull/979